kde-plasma/qqc2-breeze-style/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>kde@gentoo.org</email>
+		<name>Gentoo KDE Project</name>
+	</maintainer>
+	<upstream>
+		<bugs-to>https://bugs.kde.org/</bugs-to>
```

Comment:
I think we are using the Invent bug tracker, or at least not Bugzilla. Or at least it didn't have a product the last time I checked...

Reply:
That would normally be reserved for dev bugs, not user bugs. Maybe something that needs to be requested from KDE infra.